FreydCategoriesForCAP/gap/CategoryOfRows.gi

```diff
@@ -876,6 +851,31 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_CATEGORY_OF_ROWS,
             
         end );
         
+        ##
+        AddBasisOfExternalHom( category,
```

Comment:
This is a correct but very expensive way to construct the basis. I would rather create `s * t` `HomalgInitialMatrix`, then make the matrices immutable. If this is not compilable, then we need `homalg` commands to create these matrices as immutable matrices immediately.

Reply:
Thanks for the review! Since this PR only moves existing code, I will merge it as is and open an issue for the improvements you suggest.

Reply:
I agree.